packages/functions/src/dal/AppointmentDataAccessLayer.ts

```diff
@@ -161,3 +161,14 @@ export function removeDonorFromDbAppointment(
     lastChangeType: BookingChange.CANCELLED,
   };
 }
+
+export function ConfirmArrivedFromDbAppointment(
```

Comment:
1. Start with lowercase `c`.
2. I am not sure I understand the name. What does the `from` mean? 
3. Should this be in AppointmentDAL? Why not have this util inside the handler? Is this used anywhere else?

Reply:
1.) ok
2.)  were taking the DBAppointment and adding the donateTime
3.) i followed the same logic as the cancle appointment... currently its not used anywhere but maybe in the future....